omod/src/main/webapp/portlets/flagPatientOverview.jsp

```diff
@@ -1,8 +1,22 @@
+<%@page import="org.openmrs.module.patientflags.Flag"%>
 <%@ include file="/WEB-INF/template/include.jsp"%>
 
 <openmrs:require privilege="View Patients" otherwise="/login.htm"
 	redirect="/index.htm" />
 
 <c:forEach items="${model.flags}" var="flag">
-	<span ${flag.priority != null ? flag.priority.style : ''}>${flag.localizedMessage}</span><br />
+	<span ${flag.priority.style}> 

```

Comment:
I was trying to remove that null content error. An assumption was the single quotes returned here. The code was working without null check, it automatically handles the empty or null nested property. Should I change it back?